docs/administration/configuring-replication/create-replication-endpoints.md

```diff
@@ -39,6 +39,24 @@ To replicate image repositories from one instance of Harbor to another Harbor or
    {{< note >}}
    - AWS ECR adapters should use access keys, not a username and password. The access key should have sufficient permissions, such as storage permission.
    - Google GCR adapters should use the entire JSON key generated in the service account. The namespace should start with the project ID.
+    - Quay Registry
+      - Supported scope
+         - [Quay.io](https://quay.io) (cloud version) does not allow to create namespaces (organizations) automaticaly, due to Recaptcha enabled on the Quay.io side.
+         - [RedHat Quay](https://www.openshift.com/products/quay) (enterprise on-premises version) is fully supported (tested on v3.2.0)
+         - [Project Quay](https://github.com/quay/quay) (open sourced version) is also supported theoretically, but has not been tested yet.
+      - Authorization
+         - None authorization, keep Access ID and Access Secret empty.
+         - With authorization, You don't need to input Access ID, default is `json_file`, input your Access Secret use json format, example:
+         ```
+         {
+         "oauth2\_token": "YmQZ1QZENVmOD6v9kENzmfptNVhgBuy5oVl85eGV", // optional
+         "account\_name": "jack",
+         "docker\_cli\_password": "q6NVazikNqIf4coiQ+JvV4iqiCpkNjE0DLX8ZMQuFRbkHk5iMv6/hd4WdV3W3nyX"
+         }
+         ```
+         `oauth2\_token` is required only if you want to create organization automatic, `account\_name` is your login name(why not robot account? because robot account can not access to different organization's repositories), `docker\_cli\_password` is your cli password which can be generated on quay ui page.
```

Comment:
We should make these bullet points.

Suggested wording changes

    - `oauth2\_token` is required only if you want to create an organization automatically
    - `account\_name` is your login name. Its not recommended to use a robot account because a robot account can not access different organization's repositories
    - `docker\_cli\_password` is your cli password. You can generate this on the Quay ui page.